packages/ember-routing-views/lib/views/link.js

```diff
@@ -421,17 +412,6 @@ var LinkComponent = EmberComponent.extend({
       this.set('disabled', attrs.disabledWhen);
     }
 
-    var currentWhen = attrs['current-when'];
-
-    if (attrs.currentWhen) {
-      Ember.deprecate('Using currentWhen with {{link-to}} is deprecated in favor of `current-when`.', !attrs.currentWhen);
-      currentWhen = attrs.currentWhen;
-    }
-
-    if (currentWhen) {
```

Comment:
This is a breaking change without deprecation as it exists here.

Consider the following usage:

```javascript
{{link-to 'stuff' 'stuff' current-when='foo'}}
```

In the above invocation, `attrs['current-when']` was previously setting `currentWhen` so that you could call `this.get('currentWhen')` (instead of `this.get('current-when')`).  We could add a `deprecatingAlias` from `currentWhen` to `current-when` if we need to...